### PR TITLE
Feature/translate sponsor btn

### PR DESCRIPTION
**Pull request by JamirLeal**
### What does this PR do?

- This PR adds a change language button in contact page at the same level of sponsor us button, that on click set the state of the language.

#### Related issues

#5

#### Merge Checklist:

- [x] The branch is up-to-date (i.e. rebased) with the base branch
- [x] No code style issues

### CODE REVIEW

File: src/components/Contact/ContactSponsorUs/ContactSponsorUs.js [hunk 1]
```diff
@@ -1,20 +1,42 @@
...
+      /** @type { string } */
+      sponsorButtonLabel: 'Sponsor us!'
+    };
   }
 
   sponsorUsCallback() {
     window.open(this.sponsorsData.url_contact, '_blank');
   }
 
+  translate() {
```
Comment (RicardoChapaRomero) [inline src/components/Contact/ContactSponsorUs/ContactSponsorUs.js:32]: Good proposal, but I think the logic is inverted. The idea is that, if the sponsor button states `Sponsor Us!` then the translate button will state `Traducir a español`. So if one button is in Spanish then the other is in English to clarify the translation option. Does that make sense?

File: src/components/Contact/ContactSponsorUs/ContactSponsorUs.js [hunk 2]
```diff
@@ -1,20 +1,42 @@
...
+    if(this.state.language === 'en') {
+      this.setState({ language: 'es', translateLabel: 'Traducir a inglés', sponsorButtonLabel: 'Patrocinanos!'});
```
Comment (RicardoChapaRomero) [inline src/components/Contact/ContactSponsorUs/ContactSponsorUs.js:34]:
```suggestion
      this.setState({ language: 'es', translateLabel: 'Traducir a inglés', sponsorButtonLabel: 'Patrocinanos!' });
```

File: src/components/Contact/ContactSponsorUs/ContactSponsorUs.js [hunk 3]
```diff
@@ -1,20 +1,42 @@
...
+    } else {
+      this.setState({ language: 'en', translateLabel: 'Translate to spanish', sponsorButtonLabel: 'Sponsor Us!'});
```
Comment (RicardoChapaRomero) [inline src/components/Contact/ContactSponsorUs/ContactSponsorUs.js:36]:
```suggestion
      this.setState({ language: 'en', translateLabel: 'Translate to spanish', sponsorButtonLabel: 'Sponsor Us!' });
```

File: src/components/Contact/ContactSponsorUs/ContactSponsorUs.css [hunk 4]
```diff
@@ -10,18 +10,36 @@
...
   color: white;
   transition: 0.3s;
   background-color: #13191E;
 }
 
-@media (max-width: 750px) {
+.contact-translate-sponsor-us-btn {
+  border: 1px solid #4F72B8;
+  border-radius: 8px;
+  font-size: 20px;
```
Comment (RicardoChapaRomero) [inline src/components/Contact/ContactSponsorUs/ContactSponsorUs.css:24]: NIT: According to Googles' HTML/CSS Code Style guide, CSS attributes should be listed alphabetically.

**Review by RicardoChapaRomero — CHANGES_REQUESTED** [review]
Is looking good. I addressed some comments

**Comment by JamirLeal** [comment]
Changes done!

**Review by RicardoChapaRomero — APPROVED** [review]
LGTM!

**Pull request merged**